Makefile

```diff
@@ -1,7 +1,7 @@
 # Makefile
-IMAGE_NAME = appimage-builder-oak-d
-CONTAINER_NAME = appimage-builder-oak-d
-AARCH64_APPIMAGE_NAME = viam-camera-oak-d--aarch64.AppImage
+IMAGE_NAME = appimage-builder-oak
```

Comment:
not in this pr - viam as a company is trying to move away from app-image architectures in favour of static builds. We can deal with this module another day, but for new modules we prefer not going the appimage route if possible.